modules/apps/frontend-theme/frontend-theme-classic/src/css/_clay_variables.scss

```diff
@@ -601,16 +617,129 @@ $btn-link: map-deep-merge(
 			color: $c-unset,
 			text-decoration: none,
 		),
-		font-weight: $font-weight-normal,
-		text-decoration: $link-decoration,
 		focus: (
 			box-shadow: $c-unset,
 			text-decoration: $link-decoration,
 		),
+		font-weight: $font-weight-normal,
 		hover: (
 			color: var(--btn-link-hover-color, $link-hover-color),
 			text-decoration: $link-hover-decoration,
 		),
+		text-decoration: $link-decoration,
 	),
 	$btn-link
 );
+
+// Utilities
+
+$bg-theme-colors: () !default;
+$bg-theme-colors: map-deep-merge(
+	(
+		'danger': (
+			background-color: var(--danger, $danger) !important,
+		),
```

Comment:
I think this can work as a temporary solution. I saw that adding custom properties directly to the `$theme-colors` breaks other components, but I believe, in the long run that's the correct change.

Reply:
@edalgrin `$theme-colors` is useless. I plan on removing this map in future versions. Its only use right now is generating the primary through dark color custom properties. I don't think Clay CSS should be generating any custom properties anyway.